.github/workflows/desktop.release.reuse.yml

```diff
@@ -78,21 +78,25 @@ jobs:
 
       - run: yarn install --immutable
 
-      - name: Rebuild Electron Native Modules
-        run: yarn workspace @standardnotes/desktop rebuild:home-server
-
-      - run: yarn build:desktop
-
       - run: echo APP_VERSION=$(node -p "require('./../web/package.json').version") >> $GITHUB_ENV
-
-      - name: Compile for AppImage
-        run: yarn run webpack --config desktop.webpack.prod.js
       - name: AppImage
-        run: yarn run electron-builder --linux --x64 -c.linux.target=AppImage --publish=never --c.extraMetadata.version=${{ env.APP_VERSION }}
+        run: |
```

Comment:
The native modules are currently being rebuilt once before the packaging for both architectures happens. I suspect that might be the reason the native modules are always x64 and the arm64 Linux packages aren't running properly (see https://github.com/standardnotes/forum/issues/3457)